terms-conditions.ftd

```diff
@@ -4,6 +4,7 @@
 
 
 -- lib.page:
+active-page: terms-conditions/
```

Comment:
@priyanka9634 as mentioned above comment - plz try to avoid additional properties to `page` component. Try a different approach.

Reply:
To achieve this fixes required in sitemap and toc processor.